src/terminal_ui.cc

```diff
@@ -1250,9 +1250,13 @@ void TerminalUI::set_resize_pending()
 
 void TerminalUI::setup_terminal()
 {
+    // enable alternative screen buffer
     fputs("\033[?1049h", stdout);
+    // enable focus notify
     fputs("\033[?1004h", stdout);
+    // hide cursor
     fputs("\033[?25l", stdout);
+    // set special keypad mode
```

Comment:
This is usually (i.e. in the original DEC terminal documentation) known as "application keypad" mode. If we're being explanatory, we might as well go all the way:
```suggestion
    // set application keypad mode, so the keypad keys send unique codes
```